### PR TITLE
Adding sector and circle code in registration table.

### PR DESCRIPTION
**Pull request by Satendra-SR**
#316 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

- As per requirement we need to add `sector` and `circle_code` column in registration table. So, updated the table structure in the registration table model.

- Query to create new columns in registration table in database:
To add **sector** column.
```sql
ALTER TABLE registration ADD sector VARCHAR(50);
```
To add **circle_code** column.
```sql
ALTER TABLE registration ADD circle_code INTEGER;
```

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: api/models/registration.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ class RegistrationStatus(object):
     has_received_callback = db.Column(db.Boolean)
     status = db.Column(db.String(50))
     signup_date = db.Column(db.DateTime)
+    sector = db.Column(db.String(50), nullable=True)
+    circle_code = db.Column(db.Integer, nullable=True)
```
Comment (GauravGusain98) [inline api/models/registration.py:59]: It should be string

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: api/models/registration.py (hunk 1 above)
Comment (Satendra-SR) [reply api/models/registration.py]: Okay.

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**